pkg/deployments/action-ids/avalanche/action-ids.json

```diff
@@ -0,0 +1,151 @@
+        "withdraw(uint256)": "0x2946cd1bba50d122afcd4e50cb175bc14fd55a6df2e8ab74458327704b001e25",
+        "withdraw(uint256,bool)": "0x5fab59a064ea7fa938816c09376c09df37fa7a684d8e9f0af2b575c326a21f23"
+      }
+    },
+    "ChildChainStreamer": {
+      "useAdaptor": true,
+      "actionIds": {
+        "add_reward(address,address,uint256)": "0xfd0982f03ae2f1b60e1a94dafa009aa791a8776bf56f7dcedd168a018b6a552f",
+        "get_reward()": "0x7ef0e867b22d9e0f26c12180ace9abda1bf81d6db74aae757d5810f8732e7f1e",
+        "initialize(address)": "0x9e7d1d7251d55fd618b3aa4a45df409c0772e5815f350ccdc477ff1f138e5aa4",
+        "notify_reward_amount(address)": "0x441f32030c75b51777e821d835e735a0bf9e2bb62d160680ef57e64c229218b1",
+        "remove_reward(address,address)": "0x6ff12a4164858fcba31510d8576d50825689dfa90773f37a08eec0bc8d7b888c",
+        "set_reward_distributor(address,address)": "0xce713085e9498c958964f4c64979785027e7a93323b68bca8e87296c696fa643",
+        "set_reward_duration(address,uint256)": "0x02c8d50e13f8c642176dd11071045c7954b0c485674723e03718910be72529fd"
+      }
+    },
+    "ChildChainLiquidityGaugeFactory": {
+      "useAdaptor": true,
+      "actionIds": {
+        "create(address)": "0xc9dc065d0516a896cd89386058734c4819fe89854b021f97724dcd1ecbc72d8b"
+      }
+    }
+  },
+  "20220517-protocol-fee-withdrawer": {
+    "ProtocolFeesWithdrawer": {
+      "useAdaptor": false,
+      "actionIds": {
+        "allowlistToken(address)": "0x1deb6bc68439c92b8383decd20cc8e1d54bc3a344bd5ea32c71ffc7c9c342e9b",
+        "denylistToken(address)": "0xf3eadfc5e658bb5d9308d4d7d2b30766bd48e5db7a3637a92428d39fb6b97645",
+        "withdrawCollectedFees(address[],uint256[],address)": "0x9b7cac7a6e50200d3ceb1372c25ac2f697fa4ba4495fb37091ffc35e5b4f8a42"
+      }
+    }
+  },
+  "20220527-child-chain-gauge-token-adder": {
+    "ChildChainGaugeTokenAdder": {
+      "useAdaptor": false,
+      "actionIds": {
+        "addTokenToGauge(address,address,address)": "0x49e783a1aedce49dedc1f4599f898c3136453816218ab89c029423bf0a25a47a"
+      }
+    }
+  },
+  "20220725-protocol-fee-percentages-provider": {
+    "ProtocolFeePercentagesProvider": {
+      "useAdaptor": false,
+      "actionIds": {
+        "registerFeeType(uint256,string,uint256,uint256)": "0x87608416e0cc3c0a215423f5fe19d9cfaaf80cbfc752636fe834fc0a8e65f11c",
+        "setFeeTypePercentage(uint256,uint256)": "0xf966da38fb72cb10be01d2ff406b1aeb9cdc62f757d0042e6ff6249f196221a1"
+      }
+    }
+  },
+  "20221123-pool-recovery-helper": {
+    "PoolRecoveryHelper": {
+      "useAdaptor": false,
+      "actionIds": {
+        "addPoolFactory(address)": "0x90ce8036a67bbe2451d4f50ab7353343339db901728ae2de09928006819ee686",
+        "enableRecoveryMode(address)": "0xd803b6aadd1ec41e42e680b246ebc3f68425ee6cfba1883b53b8fd01e7f37c6b",
+        "removePoolFactory(address)": "0x25cbdd96392463c2c0b1e5a2db364a18327ef561b6ceeb4a444b694dbf2c7890"
+      }
+    }
+  },
+  "20221124-authorizer-adaptor-entrypoint": {
```

Comment:
Looks like we're missing the action-ids for the regular authorizer-adaptor

Reply:
Actually neither is permissioned - this entry shouldn't be here

Reply:
Should they be removed from all the other networks too, then? (They just have "performAction".) All of them have entries for both.

Reply:
It doesn't hurt to have extra stuff here, but sure, go for it. We actually have lots of entries in these jsons that don't do anything (e.g. for functions that are not permissioned).

You could also add it to `ignored-functions.json`.